Frontend/src/app/auth/pages/login/login.component.html

```diff
@@ -30,7 +30,16 @@ <h4 class="text-black">Login</h4>
           <p class="error" *ngIf="error">{{ error }}</p>
 
           <div class="form-group">
-            <button nbButton fullWidth status="success" (click)="login(loginForm.value)">Login</button>
+            <button 
+              nbButton 
+              fullWidth 
+              status="success" 
+              (click)="login(loginForm.value)"
+              [nbSpinner]="isLoading"
+              [disabled]="isLoading"
+              nbSpinnerStatus="danger">
```

Comment:
I feel like green would look better here than red, what do you think? Red is a scary color.